hak fetch tries to download devDependencies for two modules. Unfortunately it does so in a very non-deterministic manner (no shrinkwrap file)
Remove that step since the devDependencies are not actually needed for any of these two modules.

--- element-desktop-1.11.86/scripts/hak/fetch.ts	2024-11-19 22:43:48.895355949 +0100
+++ element-desktop-1.11.86/scripts/hak/fetch.ts	2024-11-19 22:44:02.245421312 +0100
@@ -26,14 +26,6 @@
     console.log("Fetching " + moduleInfo.name + "@" + moduleInfo.version);
 
     const packumentCache = new Map();
-    await pacote.extract(`${moduleInfo.name}@${moduleInfo.version}`, moduleInfo.moduleBuildDir, {
-        packumentCache,
-    });
-
-    console.log("Running yarn install in " + moduleInfo.moduleBuildDir);
-    await hakEnv.spawn("yarn", ["install", "--ignore-scripts"], {
-        cwd: moduleInfo.moduleBuildDir,
-    });
 
     // also extract another copy to the output directory at this point
     // nb. we do not yarn install in the output copy: we could install in
